src/main/java/picard/vcf/VcfUtils.java

```diff
@@ -0,0 +1,26 @@
+package picard.vcf;
+
+import java.io.File;
+
+/**
+ * Created by farjoun on 4/1/17.
+ */
+public class VcfUtils {
+
+    static String UNCOMPRESSED_VCF_ENDING=".vcf";
+    static String COMPRESSED_VCF_ENDING=".vcf.gz";
+    static String BCF_ENDING=".bcf";
+
+    /**
+     * Checks if the suffix is one of those that are allowed for the various
+     * formats that contain variants (currently vcf and bcf)
+     */
+    static public boolean isVariantFile(final File file){
+        final String name = file.getName();
+
+        if (name.endsWith(UNCOMPRESSED_VCF_ENDING)) return true;
```

Comment:
ah.. didn't see the endsWith..

perhaps simplify it to 

`return name.endsWith(UNCOMPRESSED_VCF_ENDING) || name.endsWith(COMPRESSED_VCF_ENDING) || name.endsWith(BCF_ENDING);`